### PR TITLE
move api-xml-adjuster from tools to build-tools.

### PR DESCRIPTION
**Pull request by atsushieno**
It is build-time tool and should not be used in any production environment.

### CODE REVIEW

File: Xamarin.Android.sln [hunk 1]
```diff
@@ -101,12 +99,16 @@ Project("{9344BDBB-3E7F-41FC-A0DD-8665D75EE146}") = "dependencies", "build-tools
...
 EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "api-xml-adjuster", "build-tools\api-xml-adjuster\api-xml-adjuster.csproj", "{8A6CB07C-E493-4A4F-AB94-038645A27118}"
+EndProject
 Global
 	GlobalSection(SolutionConfigurationPlatforms) = preSolution
 		Debug|AnyCPU = Debug|AnyCPU
 		Release|AnyCPU = Release|AnyCPU
 		XAIntegrationDebug|AnyCPU = XAIntegrationDebug|AnyCPU
 		XAIntegrationRelease|AnyCPU = XAIntegrationRelease|AnyCPU
+		XAIntegrationDebug|Any CPU = XAIntegrationDebug|Any CPU
```
Comment (jonpryor) [inline Xamarin.Android.sln]: `msbuild` doesn't like this and the following line. Results in:

    error MSB4126: The specified solution configuration "Debug|Any CPU" is invalid. Please specify a valid solution configuration using the Configuration and Platform properties (e.g. MSBuild.exe Solution.sln /p:Configuration=Debug /p:Platform="Any CPU") or leave those properties blank to use the default solution configuration.

Yes, this line and the following line are added by Xamarin Studio/MonoDevelop. Yes, they're broken. :-(

File: Xamarin.Android.sln [hunk 2]
```diff
@@ -467,6 +461,18 @@ Global
...
 		{4B9D96BB-95AB-44E8-9F87-13B12C8BCED1}.XAIntegrationRelease|AnyCPU.Build.0 = Debug|Any CPU
+		{8A6CB07C-E493-4A4F-AB94-038645A27118}.Debug|AnyCPU.ActiveCfg = Debug|Any CPU
+		{8A6CB07C-E493-4A4F-AB94-038645A27118}.Debug|AnyCPU.Build.0 = Debug|Any CPU
+		{8A6CB07C-E493-4A4F-AB94-038645A27118}.Release|AnyCPU.ActiveCfg = Release|Any CPU
+		{8A6CB07C-E493-4A4F-AB94-038645A27118}.Release|AnyCPU.Build.0 = Release|Any CPU
+		{8A6CB07C-E493-4A4F-AB94-038645A27118}.XAIntegrationDebug|AnyCPU.ActiveCfg = Debug|Any CPU
+		{8A6CB07C-E493-4A4F-AB94-038645A27118}.XAIntegrationDebug|AnyCPU.Build.0 = Debug|Any CPU
+		{8A6CB07C-E493-4A4F-AB94-038645A27118}.XAIntegrationRelease|AnyCPU.ActiveCfg = Debug|Any CPU
+		{8A6CB07C-E493-4A4F-AB94-038645A27118}.XAIntegrationRelease|AnyCPU.Build.0 = Debug|Any CPU
+		{8A6CB07C-E493-4A4F-AB94-038645A27118}.XAIntegrationDebug|Any CPU.ActiveCfg = Debug|Any CPU
```
Comment (jonpryor) [inline Xamarin.Android.sln]: This and the following three lines -- those with `Any CPU` (note the space) -- should be removed.

File: Xamarin.Android.sln (hunk 1 above)
Comment (atsushieno) [inline Xamarin.Android.sln]: eww.

Is there any reason we have those "Any CPU" profiles in the GlobalSection at the top of this file?

Comment (atsushieno) [inline Xamarin.Android.sln]: Never mind, I misread the file.

**Pull request merged**